xml/System.IO/EnumerationOptions.xml

```diff
@@ -99,7 +99,7 @@
         <ReturnType>System.IO.FileAttributes</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets the attributes to skip.</summary>
+        <summary>Gets or sets the attributes to skip. The default is <c>FileAttributes.Hidden | FileAttributes.System</c>.</summary>
```

Comment:
What's the purpose of moving the defaults to the summary? According to [our API reference guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value), the default belongs in \<value>.

Reply:
When looking at the [page for the class `EnumerationOptions`](https://learn.microsoft.com/en-us/dotnet/api/system.io.enumerationoptions?view=net-7.0), the constructor summary says "with the recommended default options" which aren't listed on that page. Clicking 16 times to figure out what the default options are does not seem like a good user experience.

Reply:
@gewarren OP wants the defaults to be shown on the [Properties](https://review.learn.microsoft.com/en-us/dotnet/api/system.io.enumerationoptions?view=net-8.0&branch=pr-en-us-9105#properties) table. An alternative could be to write them down in the remarks section of the type.

Reply:
Perhaps it makes most sense to document the default values in the remarks of the constructor - https://review.learn.microsoft.com/en-us/dotnet/api/system.io.enumerationoptions.-ctor?view=net-8.0&branch=pr-en-us-9105#system-io-enumerationoptions-ctor. @NetMage Does that work for you? That way we can just keep the default values in the Property Value section of the property pages, which matches the guidelines.